tests/unit/test_parameterisation.py

```diff
@@ -0,0 +1,45 @@
+import pybop
+import pytest
+import numpy as np
+import pandas as pd
+
+
+class TestParameterisation:
+    def test_rmse(self):
+        # Form observations
+        Measurements = pd.read_csv("examples/Chen_example.csv", comment='#').to_numpy()
```

Comment:
using data files can be annoying if you try to run the tests from a different directory. Could we just generate some simulation data on the fly instead?

Reply:
Fair shout, this should be pretty easy to solve via importing pybamm and grabbing sample data from modified parameter simulation object.

Reply:
try not to assume you are using pybamm too widely. simulating from a model will be a pretty common thing to do, so it might be useful to have the funcionality in the model class (or if it doesn't fit there a separate Simulation or Solver class)